packages/airnode-deployer/src/utils/logger.ts

```diff
@@ -27,15 +33,22 @@ function oraInstance(text?: string) {
   return debugModeFlag ? ora.default({ text, prefixText: () => new Date().toISOString() }) : ora.default(text);
 }
 
+export function writeLog(text: string) {
+  fs.appendFileSync('config/deployer-log.log', `${new Date(Date.now()).toISOString()}: ${text}\n`);
+}
+
 export function succeed(text: string) {
+  writeLog(text);
   oraInstance().succeed(text);
 }
 
-export function fail(text: string) {
-  oraInstance().fail(text);
+export function fail(text: string, options: LoggerOptions = { bold: false }) {
+  writeLog(text);
+  oraInstance().fail(options.bold ? bold(text) : text);
```

Comment:
I had to move the use of `bold` here because it created some weird characters when saved to a file